### DISCUSSION TITLE
Welcome to CodeFactory Discussions!

### DISCUSSION THREAD

**Discussion by jhannahLCD** [CodeFactoryLLC/CodeFactory#1] [Support] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arvind12mmmec** [comment]
Hi I have almost 20 VB.NET Winform APPs that I need to migrate to MAUI... Is Code factory is fine enough to convert Form/Screen & code to a basic MAUI so that we can at-least reduce some overhead... Later We will use API's instead of direct call to DB...

Out Structure/Tech-Stack is as follows:

1- Winforms (VB.NET) 
2- Framework mostly used is 4.6
3- SQL Server direct call to DB using ADODB & OLEDB
4- SAP Crystal Reports

**Reply by bfencken** [reply]
Feel free to drop our team an email at support@codefactory.software and we can recommend some options.

**Reply by arvind12mmmec** [reply]
Thanks, I have sent an email with my concerns.

**Reply by bfencken** [reply]
Didn't receive your email. Did you send it from your work account?

**Reply by bfencken** [reply, marked as answer]
Hi Arvind,
Unfortunately, VB.net is not supported by CodeFactory. I did get your email. Thanks for reaching out.
